pywemo/util.py

```diff
@@ -69,11 +70,25 @@ def interface_addresses() -> list[str]:
     return addresses
 
 
-def get_ip_address(host: str = '1.2.3.4') -> str | None:
-    """Return IP from hostname or IP."""
+def get_callback_address(
+    device_host: str,
+    callback_port: int,
+    callback_address: str | None = None,
```

Comment:
Let's drop the `callback_address` parameter. I think plumbing it through as an argument is adding complexity. Let's go with returning it PYWEMO_CALLBACK_ADDRESS if it exists, or the socket/connect/getsockname logic if it is unset.